core/templates/components/summary-tile/lesson-card.component.ts

```diff
@@ -0,0 +1,179 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for a lessons in learner dashboard
```

Comment:
lessons->lesson

Is it only for the learner dashboard? If not, drop the "in learner dashboard". If it's only for the learner dashboard, then in->on

Reply:
Done.